scripts/mkcloud

```diff
@@ -1084,8 +1084,6 @@ Optional
     scenario_dir='/tmp' (default=${SCRIPTS_DIR}/scenarios/cloud\$(getcloudver)/)
         Full path to directory containing scenario file.
 EOUSAGE
-    # Source qa_crowbarsetup.sh, which is where onadmin_help lives
-    source ${SCRIPTS_DIR}/qa_crowbarsetup.sh
```

Comment:
Well, that's kind of what I meant :smile: Printing 'no file or directory' for the help function breaks the mkcloud though. You can't call the help function anymore.